infra/gp-multena/values.yaml

```diff
@@ -0,0 +1,28 @@
+multena:
+  dev: false
+  logLevel: DEBUG
+  adminGroup: Gepardec
+  jwksCertUrl: https://sso.apps.c-gepa-play.gepa.vshnmanaged.net/realms/internal/protocol/openid-connect/certs
```

Comment:
keine fixen URLs in den beispiel values.

Reply:
kommt man da vielleicht über den keycloak service hin? dann wär das nochmal schöner.